app/api-fetch/artist-by-id.ts

```
@@ -32,12 +30,28 @@ const ArtistSchema = z.object({
  */
 export default async function fetchArtistById(id: string) {
     console.log(`Fetching artist with ID: ${id}`);
-
     try {
-        const response = await axios.get(`${process.env.NEXT_PUBLIC_API_URL}/v1/artist/${id}`);
-        const data = ArtistSchema.parse(response.data);
-        return data["data"][0] as Artist;
+        if (localStorage.getItem("artist-" + id) !== null && localStorage.getItem("artistTime-" + id) === null) {
+            localStorage.removeItem("artist-" + id);
+        }
+        if (localStorage.getItem("artist-" + id) !== null || Date.now() - parseInt(localStorage.getItem("artistTime-" + id)!) < 3600000) {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)